src/main/java/com/esri/samples/raster/raster_rendering_rule/RasterRenderingRuleSample.java

```diff
@@ -0,0 +1,166 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.samples.raster.raster_rendering_rule;
+
+import java.util.List;
+
+import com.esri.arcgisruntime.arcgisservices.RenderingRuleInfo;
+import com.esri.arcgisruntime.layers.RasterLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.ImageServiceRaster;
+import com.esri.arcgisruntime.raster.RenderingRule;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.ComboBox;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+public class RasterRenderingRuleSample extends Application {
+
+    private ComboBox<String> renderingRuleDropDownMenu;
+    private MapView mapView;
+    private ArcGISMap map;
+    private final String imageServiceUrl = "http://sampleserver6.arcgisonline.com/arcgis/rest/services/CharlotteLAS/ImageServer";
+
+    @Override
+    public void start(Stage stage) {
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Raster Rendering Rule Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // create drop down menu of Rendering Rules
+            renderingRuleDropDownMenu = new ComboBox<>();
+            renderingRuleDropDownMenu.setPromptText("Select a Raster Rendering Rule");
+            renderingRuleDropDownMenu.setEditable(false);
+            renderingRuleDropDownMenu.setMaxWidth(260.0);
+
+            // create a Streets BaseMap
+            map = new ArcGISMap(Basemap.createStreets());
+
+            // add the map to a new map view
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // create an Image Service Raster as a raster layer and add to map
+            final ImageServiceRaster imageServiceRaster = new ImageServiceRaster(imageServiceUrl);
+            final RasterLayer imageRasterLayer = new RasterLayer(imageServiceRaster);
+            map.getOperationalLayers().add(imageRasterLayer);
+
+            imageServiceRaster.loadAsync();
```

Comment:
I don't think this is needed. The map will auto-load the layer, and the layer will auto-load the raster.